apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -387,5 +397,111 @@ export function createBookingPageFixture(page: Page) {
       await scheduleSuccessfullyPage.waitFor({ state: "visible" });
       await expect(scheduleSuccessfullyPage).toBeVisible();
     },
+    createTeam: async (name: string) => {
+      const teamsText = (await localize("en"))("teams");
+      const continueText = (await localize("en"))("continue");
+      const publishTeamText = (await localize("en"))("team_publish");
+
+      await page.getByRole("link", { name: teamsText }).click();
+      await page.getByTestId("new-team-btn").click();
+      await page.getByPlaceholder("Acme Inc.").click();
+      await page.getByPlaceholder("Acme Inc.").fill(`${name}-${randomString(3)}`);
+      await page.getByRole("button", { name: continueText }).click();
+      await page.getByRole("button", { name: publishTeamText }).click();
+
+      await page.getByTestId("vertical-tab-Back").click();
+    },
+    createTeamEventType: async (name: string, options: teamBookingtypes) => {
+      await page.getByTestId("new-event-type").click();
+      await page.getByTestId("option-0").click();
+
+      // We first simulate to create a default event type to check if managed option is not available
+
+      const managedEventDescription = (await localize("en"))("managed_event_description");
+      const roundRobinEventDescription = (await localize("en"))("round_robin_description");
+      const collectiveEventDescription = (await localize("en"))("collective_description");
+      const quickChatText = (await localize("en"))("quick_chat");
+      await expect(page.locator("div").filter({ hasText: managedEventDescription })).toBeHidden();
```

Comment:
why do we need to check this in createTeamEventType fixture?

Reply:
we are using this to ensure we can't see the option to create a managed event type if we don't have a team, in this case, we should see this:

<img width="569" alt="image" src="https://github.com/calcom/cal.com/assets/121884634/29940f09-59a5-4d8c-8f1e-318746186696">

And we should see this if we have a created team:

<img width="553" alt="image" src="https://github.com/calcom/cal.com/assets/121884634/493cdc04-3618-49dd-801e-68091127fd7c">